blocks-config/forms/class-uagb-forms.php

```diff
@@ -57,6 +57,71 @@ public function __construct() {
 
 		}
 
+		/**
+		 * Return the blocks content for reusable block.
+		 *
+		 * @param int $reusable_ref_id reference id of reusable block.
+		 * @since 2.6.2
+		 * @return array
+		 */
+		public function reusable_block_content_on_page( $reusable_ref_id ) {
+			if ( is_int( $reusable_ref_id ) ) {
+				$content                = get_post_field( 'post_content', $reusable_ref_id );
+				$reusable_block_content = parse_blocks( $content );
+				return $reusable_block_content;
```

Comment:
```suggestion
				$content                = get_post_field( 'post_content', $reusable_ref_id );
				return parse_blocks( $content );
```
Remove unwanted variable.

Reply:
Done here - https://github.com/brainstormforce/ultimate-addons-for-gutenberg/pull/3213. Please review.